src/skyzone_msp.cpp

```diff
@@ -114,9 +111,24 @@ SkyzoneMSP::SendRecordingState()
     msp.sendPacket(&packet, m_port);
 }
 
+void
+SkyzoneMSP::SendHeadTrackingEnableCmd(bool enable)
```

Comment:
The skyzones dont support HT do they?

Reply:
I've forwarded the updated document to skyzone so they can add it if they like. It should be possible.

Reply:
They'd need to add an IMU though right?
Which means new hardware?
I guess I'm ok to leave it "in case they do", but it seems a bit weird sending the Skyzone module an MSP message that it doesn't support.

Reply:
No, the skyzones (04X) have an IMU already built in and have a head-tracker port on the bottom, so it should kust be an update to their firmware.

Reply:
Oh damn! Didn't even know that! Lucky I don't own a set or that'd be embarrassing... oh wait.